sphinx_wagtail_theme/theme.conf

```diff
@@ -12,5 +12,5 @@ logo_height = 59
 logo_url = /
 logo_width = 45
 header_links =
-footer_links = Features|https://wagtail.io/features/, About Wagtail|https://wagtail.io/about-wagtail/, Services|https://wagtail.io/services/, Blog|https://wagtail.io/blog/, Packages|https://wagtail.io/packages/, Developers|https://wagtail.io/developers/
+footer_links = Features|https://wagtail.org/features/, About Wagtail|https://wagtail.org/about-wagtail/, Services|https://wagtail.org/services/, Blog|https://wagtail.org/blog/, Packages|https://wagtail.org/packages/, Developers|https://wagtail.org/developers/
```

Comment:
I would think the default here should be empty, and move these footer_links to Wagtail core's `docs/conf.py`